clients/client-go/README.md

```diff
@@ -641,7 +657,7 @@ func main() {
 	fmt.Println("")
 	fmt.Printf("curl -L https://queue.taskcluster.net/v1/task/%v/runs/0/artifacts/public/logs/live.log | gunzip\n", taskID)
 }
-```
+````
```

Comment:
There are a lot of unrelated changes in this README -- and this one in particular looks incorrect.  Why are they here?

Reply:
Oh, sorry. It must be a typo from my side.

Reply:
Is there a way to log the `apiDefs` inside `GenerateCode `. I tried ` fmt.Println(apiDefs[i])`. It did not work.